packages/app/cypress/e2e/support/execute-spec.ts

```diff
@@ -17,17 +17,20 @@ declare global {
        * 3. Waits (with a timeout of 30s) for the Rerun all tests button to be present. This ensures all tests have completed
        *
        */
-      waitForSpecToFinish(expectedResults?: ExpectedResults, timeout?: number): void
+      waitForSpecToFinish(expectedResults?: ExpectedResults, timeout?: number, checkForStats?: boolean): void
       verifyE2ESelected(): void
       verifyCtSelected(): void
     }
   }
 }
 
-export const waitForSpecToFinish = (expectedResults, timeout?: number) => {
-  // First ensure the test is loaded
-  cy.get('.passed > .num').should('exist')
-  cy.get('.failed > .num').should('exist')
+export const waitForSpecToFinish = (expectedResults, timeout?: number, checkForStats = true) => {
+  // If we're not in a single studio test, we won't have the stats so we can skip this
+  if (checkForStats) {
```

Comment:
Instead of passing in the `checkForStats` parameter, can we just check if the studio panel is open to make the determination ourselves.